d2common/d2fileformats/d2mpq/mpq.go

```diff
@@ -201,5 +200,5 @@ func openIgnoreCase(mpqPath string) (*os.File, error) {
 		}
 	}
 
-	return os.Open(path.Join(mpqDir, mpqName)) //nolint:gosec // Will fix later
+	return os.Open(filepath.Join(mpqDir, mpqName)) //nolint:gosec // Will fix later
```

Comment:
could you also fix this nolint directive: https://stackoverflow.com/questions/52320708/how-to-handle-gosec-linter-warning-potential-file-inclusion-via-variable